SFBaseKit/Utils/Observable.swift

```diff
@@ -32,12 +32,70 @@ public final class Observable<T> {
     }
     
     /// Attaches a closure to execute when the binding value changes.
-    /// - Parameter receiveValue: The closure to execute with the new value.
+    /// The closure is executed on attachment.
+    /// - Parameters:
+    ///   - dispatchQueue: dispatchQueue on which to receive elements from the publisher.
```

Comment:
[optional]

Maybe add "Default is `main`." at the end of the line.

P.S. Similar for any parameter with a default value,

Reply:
Added where applicable.